### PR TITLE
FEATURE: SD-405 upgrade highcharts 7.1.1

### PR DESCRIPTION
**Pull request by lanhq**
[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs
- gdc-app-components: TBD
- gdc-analytical-designer: TBD
- gdc-dashboards: TBD

# Related Jira tasks
- SD-405: https://jira.intgdc.com/browse/SD-405

### CODE REVIEW

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3449/ : FAILURE in 7m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4275/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1181/ : FAILURE in 20m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4276/ : SUCCESS in 12s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3461/ : SUCCESS in 8m 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4284/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1184/ : FAILURE in 28m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4285/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3513/ : SUCCESS in 9m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4356/ : SUCCESS in 10s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3527/ : FAILURE in 7m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4374/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1222/ : FAILURE in 17m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4375/ : SUCCESS in 13s

**Comment by lanhq** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-951.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3578/ : SUCCESS in 16m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4444/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1257/ : FAILURE in 41m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4445/ : SUCCESS in 9s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3583/ : SUCCESS in 9m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4456/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1264/ : FAILURE in 17m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4457/ : SUCCESS in 9s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3585/ : SUCCESS in 9m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4460/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1266/ : FAILURE in 16m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4461/ : SUCCESS in 9s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1267/ : FAILURE in 10m 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4463/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3586/ : SUCCESS in 14m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4462/ : SUCCESS in 10s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1272/ : FAILURE in 12m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4475/ : SUCCESS in 9s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1928/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1280/ : UNSTABLE in 16m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4494/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3606/ : SUCCESS in 24m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4493/ : SUCCESS in 14s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3607/ : FAILURE in 6m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4496/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1282/ : FAILURE in 11m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4497/ : SUCCESS in 8s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3608/ : FAILURE in 6m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4498/ : SUCCESS in 7s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1931/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1283/ : UNSTABLE in 12m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4499/ : SUCCESS in 6s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3609/ : SUCCESS in 4m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4500/ : SUCCESS in 6s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1932/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1284/ : SUCCESS in 12m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4501/ : SUCCESS in 7s

**Comment by lanhq** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-951.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1285/ : FAILURE in 38m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4502/ : SUCCESS in 8s

**Comment by lanhq** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-951.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3610/ : SUCCESS in 3m 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4503/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1286/ : FAILURE in 32m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4504/ : SUCCESS in 7s

**Comment by lanhq** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-951.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3611/ : SUCCESS in 4m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4505/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1287/ : SUCCESS in 26m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4506/ : SUCCESS in 8s

File: examples/test/VisualizationByUri_test.js [hunk 1]
```diff
@@ -18,14 +18,25 @@ async function checkRenderedChart(selector, t) {
...
 }
 
+test("ddTable visualization should render", async t => {
+    const table = Selector(".s-visualization-table");
+    const tableHeader = Selector(
+        ".s-visualization-table .gd-column-group-header--first .s-header-cell-label",
+    );
+    await t
+        .expect(table.exists)
+        .ok()
```
Comment (ptquang86) [inline examples/test/VisualizationByUri_test.js]: - Use `eql(...)` instead of `ok/notOk`

File: examples/test/VisualizationByIdentifier_test.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ test("Chart visualization should render", async t => {
         .ok()
         .expect(chart.textContent)
         .eql(
-            "Created with Highcharts 4.2.5Month/Year (Date)$ Total Sales$2,707,184$2,625,617$2,579,553Jan 2016Feb 2016Mar 20160M1M2M3M",
+            "Created with Highcharts 7.1.1Month/Year (Date)$ Total Sales$2,707,184$2,625,617$2,579,553Jan 2016Feb 2016Mar 201601M2M3M",
```
Comment (ptquang86) [inline examples/test/VisualizationByIdentifier_test.js]: Should we move `Highcharts 7.1.1` into constant?

Comment (ptquang86) [reply examples/test/VisualizationByIdentifier_test.js]: or have some way to change this automatically ?

File: examples/test/VisualizationByUri_test.js [hunk 3]
```diff
@@ -18,14 +18,25 @@ async function checkRenderedChart(selector, t) {
...
+});
 test("Chart visualization should render", async t => {
     const chart = Selector(".s-visualization-chart svg"); // could need timeout ie 20 secs to work
     await t
         .expect(chart.exists)
         .ok()
         .expect(chart.textContent)
         .eql(
-            "Created with Highcharts 4.2.5Month/Year (Date)$ Total Sales$2,707,184$2,625,617$2,579,553Jan 2016Feb 2016Mar 20160M1M2M3M",
+            "Created with Highcharts 7.1.1Month/Year (Date)$ Total Sales$2,707,184$2,625,617$2,579,553Jan 2016Feb 2016Mar 201601M2M3M",
```
Comment (ptquang86) [inline examples/test/VisualizationByUri_test.js]: same as [here](https://github.com/gooddata/gooddata-react-components/pull/951/commits/aaa0788aa7876e75504e05d6ce2bb9d51435beea#r285491363)

File: src/components/visualizations/chart/highcharts/commonConfiguration.ts [hunk 4]
```diff
@@ -97,23 +102,29 @@ const BASE_TEMPLATE: any = {
...
 
-function registerLoadHandler(configuration: any, chartOptions: any) {
+export function handleChartLoad(): void {
+    setupDrilldown(this);
+}
+
+function registerRenderHandler(configuration: any, chartOptions: any) {
     if (isOneOfTypes(chartOptions.type, supportedDualAxesChartTypes)) {
-        set(configuration, "chart.events.load", handleChartLoad);
+        set(configuration, "chart.events.render", handleChartLoad);
```
Comment (naibinh) [inline src/components/visualizations/chart/highcharts/commonConfiguration.ts]: Could you please explain why we change 'load' to 'render'? Thanks

File: src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// (C) 2007-2018 GoodData Corporation
+
```
Comment (naibinh) [inline src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts]: remove this empty line please

File: src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import isNil = require("lodash/isNil");
+
+export function getBubbleRadius(Highcharts: any) {
```
Comment (naibinh) [inline src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts]: Do we have any type for `Highcharts`?

File: src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+            const [minSize, maxSize, value] = args.slice(2, 5);
+            let radius = proceed.apply(this, args);
+            let pos;
+            if (isNaN(value) && isNil(radius)) {
+                pos = Math.sqrt(0.5);
+                radius = Math.ceil(minSize + pos * (maxSize - minSize)) / 2;
+            }
+            return radius;
+        });
+    }
```
Comment (naibinh) [inline src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts]: please consider this, I want it much explicit as possible
and it's nice to have more comments about `Math.sqrt(0.5)`
thanks
```
if (Highcharts.seriesTypes.bubble) {
        wrap(Highcharts.seriesTypes.bubble.prototype, "getRadius", function(proceed: any, _zMin:number, _zMax: number, minSize: number, maxSize: number, value: number) {
            const radius = proceed.apply(this, [_zMin, _zMax, minSize, maxSize, value]);
            if (isNaN(value) && isNil(radius)) {
                const pos = Math.sqrt(0.5);
                return Math.ceil(minSize + pos * (maxSize - minSize)) / 2;
            }
            return radius;
        });
    }
```

File: src/components/visualizations/utils/drilldownEventing.ts [hunk 8]
```diff
@@ -131,13 +139,51 @@ const chartClickDebounced = debounce(
...
+const tickLabelClickDebounce = debounce(
+    (
+        drillConfig: IDrillConfig,
+        points: IHighchartsPointObject[],
+        target: EventTarget,
+        chartType: ChartType,
+    ): void => {
+        const { afm, onFiredDrillEvent } = drillConfig;
+        const contextPoints: IDrillPoint[] = points.map((point: IHighchartsPointObject) => {
+            return {
```
Comment (naibinh) [inline src/components/visualizations/utils/drilldownEventing.ts]: we can omit `return` :)

File: src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts (hunk 7 above)
Comment (lanhq) [reply src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts]: comments is added

File: src/components/visualizations/chart/highcharts/commonConfiguration.ts (hunk 4 above)
Comment (lanhq) [reply src/components/visualizations/chart/highcharts/commonConfiguration.ts]: on event load, `ddPoints` is not available on Axis. `ddPoints` is set on 'render' event. See detail [here](https://github.com/highcharts/highcharts/blob/master/js/modules/drilldown.src.js#L821)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3628/ : SUCCESS in 9m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4527/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3663/ : SUCCESS in 8m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4567/ : SUCCESS in 6s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3664/ : SUCCESS in 3m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4568/ : SUCCESS in 8s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1964/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1312/ : UNSTABLE in 13m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4569/ : SUCCESS in 11s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3665/ : SUCCESS in 8m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4570/ : SUCCESS in 8s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1965/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1313/ : UNSTABLE in 13m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4571/ : SUCCESS in 7s

**Comment by lanhq** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3666/ : SUCCESS in 3m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4572/ : SUCCESS in 9s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1966/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1314/ : SUCCESS in 11m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4573/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3670/ : SUCCESS in 7m 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4580/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3674/ : SUCCESS in 3m 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4584/ : SUCCESS in 6s

File: src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts (hunk 7 above)
Comment (lanhq) [reply src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3675/ : SUCCESS in 11m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4585/ : SUCCESS in 16s

File: src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts (hunk 7 above)
Comment (naibinh) [reply src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts]: so we should have const for 0.5 and add comment right there.
and please consider not using `let pos` anymore. We can use this constant directly.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by lanhq** [comment]
ok to test

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by lanhq** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3690/ : SUCCESS in 4m 07s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4608/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1984/ : SUCCESS in 12m 35s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3691/ : SUCCESS in 4m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/999/ : SUCCESS in 1m 04s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1113/ : SUCCESS in 31m 44s

**Pull request merged**